docs/api.md

```diff
@@ -62,6 +62,7 @@ These are the API group versions which are curently available when using Antrea.
 | `core.antrea.tanzu.vmware.com` | `v1alpha1` | No | v0.8.0 | v0.11.0 | v0.11.0 |
 | `networking.antrea.tanzu.vmware.com` | `v1beta1` | Yes | v0.3.0 | v0.10.0 | v1.2.0 |
 | `controlplane.antrea.tanzu.vmware.com` | `v1beta1` | Yes | v0.10.0 | v0.11.0 | v1.3.0 |
+| `controlplane.antrea.io` | `v1beta1` | Yes | v1.0.0 | v0.11.0 | v1.3.0 |
```

Comment:
I will need @tnqn to confirm but it seems to me that this API was never actually served, so maybe it shouldn't be part of this list.

It also doesn't make sense to have an introduction date that precedes the deprecation date.

Reply:
ah, yes, my bad, let me wait @tnqn 's confirmation to remove it or update the version.

Reply:
btw, I added the API list in `CleanupDeprecatedAPIServices` method in `pkg/apiserver/apiserver.go`,  I suppose it's safe to add it even we didn't actually serve it?`

Reply:
Yes, the version is never used and shouldn't be added.

We shouldn't add its APIService to `CleanupDeprecatedAPIServices` as we never create it. Deleting it will need delete permission of this resource name, which doesn't make sense to ask for.

Reply:
@tnqn  I just applied the change in PR #2631 , please help to review on it. thanks!